### DISCUSSION TITLE
Feature request: Allow search terms in URL

### DISCUSSION THREAD

**Discussion by RomkeVdMeulen** [tailwindlabs/tailwindcss#3110] [Ideas]
My kind regards and complements for your work. I wish I'd learned about it years ago.

I have one small request about the documentation. My preferred method of searching through doc is by assigning a search-engine shortcut key in my browser. For this to work, there has to be an URL for the browser to compose the search term into (e.g. `website.org/seach?term=%s`). If there is such an URL for the Tailwind doc, I've not found it. (For that matter: it doesn't seem to be possible to search the doc at all without Javascript enabled). Could something like this be added at some point?

**Comment by simonswiss** [comment]
Hi!

This is not something that is currently available. I can't promise that this is something that will happen in the future, but if the request becomes popular and we can justify de-prioritizing other projects to implement that, we'll consider it!

Moving this to discussions where it might catch a bit more attention 👍
